distributed/worker.py

```diff
@@ -3259,13 +3259,12 @@ async def _get_data():
 
 job_counter = [0]
 
-
 cache_loads = LRU(maxsize=100)
-
+enable_caching = dask.config.get("distributed.worker.cache", False)
 
 def loads_function(bytes_object):
     """ Load a function from bytes, cache bytes """
-    if len(bytes_object) < 100000:
+    if len(bytes_object) < 100000 and enable_caching:
```

Comment:
Perhaps a more generic way of doing this would be to make the `100000` number configurable?  That way you could specify a value of `0` to turn off function caching, and others could use this same value to tune the size in bytes that are allowed.  If we do this I encourage you to set the config value to a string and use the `dask.utils.parse_bytes` function

Reply:
I prefer `dask.utils.parse_bytes`, so we can set to 0 to disable it.